lesson04/assignment/find.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <script src="https://kit.fontawesome.com/e418cb4dcb.js"></script>
+  <link rel="stylesheet" href="css/style.css">
+  <title>Find a swap</title>
+</head>
+<body>
+  <!--Header, to include background photo-->
+  <header>
+  </header>
+
+  <!--Navigation menu-->
+  <nav class="nav">
+    <a class="nav-link" href="index.html">Home</a> /
+    <a class="nav-link" href="tips.html">Tips & inspiration</a> /
+    Find a swap
+  </nav>
+
+  <!--First content block on find a swap page-->
+  <section class="primary">
+    <h1>Find a swap</h1>
+    <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga. Faceaquae estioria derum recuptatur, cum volore, undipsa doloreium hillupta aut es ut alitatuscit ommossum haritatur arum qui officae videbiti corporeium faccull oribus es quidignis ipietus explam sus am aut amet ant fugiatum, utem non reptat ad magnienecto volorent fugia nonsedi gendand itatquos as magnimpor aut molor autatia ad est officimus accae et omnia cus, od qui ate</p>
+  </section>
+
+  <!--Sidebar -->
+  <aside class="sidebar">
+    <h2>Who we are</h2>
+    <!--Paper airplane button links to Contact page-->
+    <!--Pulsing button with transition created by Will Woodgate, JSFiddle: https://jsfiddle.net/willwood/gjf139z4/5/-->
+    <!--I also referred to the transform code used in this Codepen by Ola: https://codepen.io/olam/pen/BkhGl?editors=1100-->
+    <div class="side-button">
+      <a title="Contact us" href="contact.html" class="hiButton"><i class="far fa-paper-plane"></i></a>
+    </div>
+    <!--Sidebar content-->
+    <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga.</p>
+    <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga.</p>
+    <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga.</p>
+    <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga.</p>
+    <p>Iduciendisite quo magnatem iuntum quid quaest ea am.</p>
+  </aside>
+ 
+  <!--Second content block on find a swap page-->
+  <article class="secondary">
+    <h2>More content</h2>
+    <p>Iduciendisite quo magnatem iuntum quid quaest ea am, tenderumet adis dolenem quidustrum fuga.</p>
+  </article>
+  
+  <!--Footer content-->
+  <footer class="footer">
+    <nav class="footer__nav">
+      <a href="index.html">Home</a> /
+      <a href="tips.html">Tips & inspiration</a> /
+      Find a swap /
+      <a href="contact.html">Contact</a>
+    </nav>
+  </footer>
+
+</body>
+</html>
```

Comment:
FYI: there are a ton of silly ipsum generators out there, if you ever want to spice up your placeholder content
https://loremipsum.io/ultimate-list-of-lorem-ipsum-generators/